pkg/reporting/exporters/markdown/markdown.go

```diff
@@ -115,7 +116,9 @@ func createFileName(event *output.ResultEvent) string {
 	filenameBuilder := &strings.Builder{}
 	filenameBuilder.WriteString(event.TemplateID)
 	filenameBuilder.WriteString("-")
-	filenameBuilder.WriteString(stringsutil.ReplaceAll(event.Matched, "_", "/", ":"))
+	filenameBuilder.WriteString(event.Host)
+	filenameBuilder.WriteString("-")
+	filenameBuilder.WriteString(uuid.NewString())
```

Comment:
is there a reason we're using UUID instead of just straight up removing the whole `stringsutil.ReplaceAll(event.Matched, "_", "/", ":")`?

Reply:
Yes. This is why we shortened it. The issue is detailed here https://github.com/projectdiscovery/nuclei/issues/5769

Reply:
I mean; removing the addition of `stringsutil.ReplaceAll` entirely rather than replacing it with a UUID. Is there any chance the filename _(or path)_ could be the same _(possibly duplicate)_?

Reply:
Got it. Yes, that's why UID.